config/initializers/npm.rb

```diff
@@ -0,0 +1 @@
+system 'npm install' if Rails.env.development? || Rails.env.test?
```

Comment:
Do we need to do something special for production?

Reply:
Good point we should change that to production if it looks like the correct way of running the command. I think this was on a guide for an app that's deployed to a heroku server. I suppose we can manage the packages manually in the development env?